packages/studio/src/components/ErrorComponentPreview.tsx

```diff
@@ -22,11 +31,27 @@ export default function ErrorComponentPreview(props: {
     setTooltipOpen(false);
   }, []);
 
-  const rect = containerRef.current?.getBoundingClientRect();
-  const tooltipPosition = rect && {
-    x: (rect.right + rect.left) / 2,
-    y: rect.top - 40,
-  };
+  document
+    .querySelector("iframe")
+    ?.contentDocument?.addEventListener("scroll", () => setTooltipOpen(false));
+
+  document.addEventListener("scroll", () => setTooltipOpen(false));
```

Comment:
can we add these listeners in a useEffect hook so that they're only added on initial render (not on each render) and so they can be removed when un-mounting the component?